templates/cluster-template-kube-vip-crs-cni.yaml

```diff
@@ -245,6 +245,7 @@ metadata:
   name: ${CLUSTER_NAME}
 spec:
   facility: ${FACILITY}
+  metro: ${METRO}
```

Comment:
[Line 161 of this template](https://github.com/kubernetes-sigs/cluster-api-provider-packet/pull/538/files#diff-48bcb4895a4f72f2a85110dcb7c18427ca2739b5f6b4db81e4d43c9e25a083f4R161) references `${FACILITY}`.  Should it be updated to also reference `${METRO}`?